src/AuxDisplay.jsx

```diff
@@ -43,12 +46,12 @@ export default class AuxDisplay extends React.Component {
         }
     }
     play() {
-        if (this.el) {
+        if (this.el && this.el.paused) {
```

Comment:
The fact that it's called so often when it shouldn't be is bad, but that's another issue that I haven't sorted out yet.